packages/component/src/SendBox/SuggestedActions.js

```diff
@@ -69,28 +62,65 @@ const SuggestedActions = ({ className, suggestedActions = [] }) => {
     />
   ));
 
-  if (suggestedActionLayout === 'stacked') {
-    return (
-      <div aria-label=" " aria-live="polite" role="status">
+  const renderSuggestedActionsEmpty = useCallback(() => <ScreenReaderText text={suggestedActionsContainerText} />, [
+    suggestedActionsContainerText
+  ]);
+
+  const renderSuggestedActionsStacked = useCallback(
+    () => (
+      <React.Fragment>
         <ScreenReaderText text={suggestedActionsContainerText} />
```

Comment:
It looks like `<ScreenReaderText text={suggestedActionsContainerText} />` is contained in `renderSuggestedActionsEmpty `, `renderSuggestedActionsStacked `, and `renderSuggestedActionsContent`. I think it would just make more sense to move it into the main `div` in the return statement. Can probably get rid of `renderSuggestedActionsEmpty `.

Reply:
`renderSuggestedActionsEmpty` is needed for accessibility.

Reply:
Why? It's just the screen reader which is identical across the three render functions you added. I think you're over complicating the `SuggestedActions` component.